kod/object/item/passitem/necklace/shadowam.kod

```diff
@@ -236,28 +230,21 @@ messages:
    RevealHiddenAttributes()
    {
       local bDone;
-
       bDone = FALSE;
 
-      % Reveal the true name and description of the item.
-      if vrName <> vrRealName
+      if pbIdentified = FALSE
       {
-         vrName = vrRealName;
-         vrDesc = vrRealDesc;
-         bDone = TRUE;
+         pbIdentified = TRUE;
+         Send(poOwner, @SomethingChanged, #what=self);
       }
-      
+
       return bDone;
```

Comment:
bDone is now always FALSE here.  I don't think that was intentional.

Reply:
Fixed.  Removed `bDone` completely since it was redundant.